test/persistence/kafka-sources/zzzz-failpint-before.td

```diff
@@ -0,0 +1,67 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+#
+# Use a failpoint to prevent persisting to disk halfway through the workload
```

Comment:
The name of this file has a typo: "failpint". Could be intentional... :sweat_smile: :beers: